mlir/utils/jenkins/Jenkinsfile

```diff
@@ -790,6 +808,20 @@ pipeline {
                             }
                         }
                     }
+                    stage("Build Composable Kernels") {
+                        steps {
+                            sh 'rm -rf composable_kernel'
+                            dir('composable_kernel') {
+                                getAndBuildCK('''
+                                    -DGPU_TARGETS="gfx1030;gfx908;gfx90a"
```

Comment:
Perhaps `-DCMAKE_BUILD_TYPE=Release` unless that's already being set above and doesn't imply `-O3` for some reason